tests/container_test.py

```diff
@@ -8,9 +8,7 @@
 # Third-Party Libraries
 import pytest
 
-ENV_VAR = "ECHO_MESSAGE"
-ENV_VAR_VAL = "Hello World from docker compose!"
-READY_MESSAGE = "This is a debug message"
+READY_MESSAGE = "listening on port"
 SECRET_QUOTE = (
     "There are no secrets better kept than the secrets everybody guesses."  # nosec
 )
```

Comment:
The `SECRET_QUOTE` should be removed, as it is now unused.

Reply:
@dav3r forgot to remove this. resolved with [4b1d0a2](https://github.com/cisagov/con-pca-tasks-docker/pull/2/commits/4b1d0a2beb4f09892a44d16eda557ace2b042b3f)